index.php

```diff
@@ -96,15 +131,29 @@
                 ?>
 
             </ul>
-            <div class="jobs-pagination-wrapper">
-                <div class="nav-links">
-                    <a class="page-numbers current">1</a>
-                    <a class="page-numbers">2</a>
-                    <a class="page-numbers">3</a>
-                    <a class="page-numbers">4</a>
-                    <a class="page-numbers">5</a>
-                </div>
-            </div>
+            <?php if (empty($_GET['search'])) {
+                //hide pagination on search
+                ?>
+                <div class="jobs-pagination-wrapper">
+                    <div class="nav-links">
+                        <?php
+                        $las_page_num = ceil(($active_jobs / JOBS_PER_PAGE));
```

Comment:
consider moving the pagination to a function, so you can use it in more than one place

Reply:
I have moved it